examples/sdk/train_api.py

```diff
@@ -0,0 +1,57 @@
+from kubeflow.training.api.training_client import TrainingClient
+from kubeflow.storage_initializer.hugging_face import (
+    HuggingFaceModelParams,
+    HuggingFaceTrainParams,
+    HfDatasetParams,
+)
+from peft import LoraConfig
+import transformers
+from transformers import TrainingArguments
+
+client = TrainingClient()
+
+client.train(
+    name="hf-test",
+    num_workers=2,
+    num_procs_per_worker=0,
```

Comment:
Why this value is 0 ?

Reply:
for cpu only training

Reply:
Hmm, but can `torchrun` be used with CPUs ?
E.g. maybe I want to run `torchrun --nproc-per-node=2` where I use 2 CPU per node.
cc @johnugeorge